Sources/Widgets/Widgets3D/ResliceCursorWidget/example/index.js

```diff
@@ -148,14 +148,14 @@ for (let i = 0; i < 3; i++) {
 // Load image
 // ----------------------------------------------------------------------------
 
-function updateReslice(viewtype, reslice, actor, renderer) {
+function updateReslice(viewtype, reslice, actor, renderer, renderwindow) {
   const modified = widget.updateReslicePlane(reslice, viewtype);
   if (modified) {
     // Get returned modified from setter to know if we have to render
     actor.setUserMatrix(reslice.getResliceAxes());
     widget.resetCamera(renderer, viewtype);
   }
-
+  renderwindow.render();
```

Comment:
This will dramatically decrease perfs. You used to not need it, what happened ?

Reply:
The display lines on the others views are not updated when I scroll on the third one, except if I move the mouse on it after scrolled.

Reply:
My bad ! I forgot to add something in the behavior.js.